internal/b/hs_with_application_service.go

```diff
@@ -0,0 +1,25 @@
+package b
+
+// BlueprintHSWithApplicationService who has an application service to interact with
+var BlueprintHSWithApplicationService = MustValidate(Blueprint{
+	Name: "alice",
+	Homeservers: []Homeserver{
+		{
+			Name: "hs1",
+			Users: []User{
+				{
+					Localpart:   "@alice",
+					DisplayName: "Alice",
+				},
+			},
+			ApplicationServices: []ApplicationService{
+				{
+					ID:              "my_as_id",
+					URL:             "http://localhost:9000",
```

Comment:
What is the purpose of this URL? Does this need to be changed when run under CI?

Reply:
It's the URL that the homeserver will callback to when it has new data to pass through it: Homeserver gets a new message -> Sends it to the application service at that address.

We're not working with any of that side of application services for my tests in https://github.com/matrix-org/complement/pull/68

This URL would probably be best implemented by a stubbed HTTP server in a Complement test which can be figured out when someone needs it.